test/cases/api.sh

```diff
@@ -145,7 +145,7 @@ do
     --cert /etc/osbuild-composer/client-crt.pem \
     https://localhost/api/composer/v1/compose/"$COMPOSE_ID")
 
-  COMPOSE_STATUS=$(echo "$OUTPUT" | jq -r '.status')
+  COMPOSE_STATUS=$(echo "$OUTPUT" | jq -r '.image_status.status')
 
   if [[ "$COMPOSE_STATUS" != "pending" && "$COMPOSE_STATUS" != "running" ]]; then
     test "$COMPOSE_STATUS" = "success"
```

Comment:
We could tes tthe upload status by adding a test here by extracting `upload_status` from the COMPOSE_STATUS here and see if it's `success` too.